ObjectiveGit/GTTree.h

```diff
@@ -64,6 +64,13 @@ typedef NS_ENUM(NSInteger, GTTreeEnumerationOptions) {
 /// returns a GTTreeEntry or nil if there is nothing with the specified name
 - (GTTreeEntry *)entryWithName:(NSString *)name;
 
+/// Get a entry by path
```

Comment:
That was actually a copy-paste error ;-) I'll fix the other occurrences as well.